.upstream-tests/test/std/atomics/atomics.lockfree/isalwayslockfree.pass.cpp

```diff
@@ -18,9 +18,13 @@
 
 #include "test_macros.h"
 
+// NVRTC doesn't include host atomic making this feature test invalid
+// TODO: Should we define __cpp_lib_atomic_is_always_lock_free for NVRTC?
```

Comment:
@griwes Food for thought regarding macro. We don't do this now for anything so it's going to have to be another PR.